File: Ruby/test2.rb
#!/home/raj/.rvm/rubies/ruby-2.3.1/bin/ruby

#(1..20).each do |i|
#	if i % 2 != 0 then
#		puts i.to_s
#	end
#end

#puts "Please enter the investment amount: "
#invAmt = gets.chomp.to_f
#puts "Please enter the interest Rate: "
#intRate = gets.chomp.to_f.round(2)


#def interest_calc(amt, rate)
#	final = 0 
#	for i in 1..10 do
#		final += amt * rate
#	end
#	final.round(2)
#end


#puts "The interest for the amount #{invAmt} with the rate #{intRate} for 10 years \
#      is #{interest_calc(invAmt,intRate)}"


puts "Please enter how tall you want your pine tree to be: 1 .. 10"

tree_height = gets.to_i

spaces = tree_height - 1

end_space = tree_height - 1

hashes = 1

while tree_height != 0 do
	
	(1..spaces).each do |x| 
		print(" ")
	end
	
	spaces -= 1
	
	
	(1..hashes).each do |y|
		print("#")
	end
	
	puts()
	
	hashes +=  2
	
	tree_height -= 1
end

(1..end_space).each do |z|
	print(" ")
end

print("#")
	


	

File: Python/test2.py
#!/home/raj/anaconda3/bin/python

#for i in range(1,21):
#	if i % 2 != 0:
#		print("{} is odd".format(i))
#	else:
#		pass

#intAmt = float ( (input("Please enter the investment Amount: ")).strip() )
#intRate = round(float ( (input("Please enter the interest Rate: ")).strip() ), 2)

#def calc_earnings(Amt, Rate):
#	result = 0
	#for i in range(1,11):
	#	result += Amt * Rate
	#return result

#print("{} amount with {} rate will yield following {} as earning".format(intAmt, intRate, calc_earnings(intAmt, intRate) ))


tree_height = int (  input("Please enter how tall you want you pine tree to be: 1 - 10: ") )

space = tree_height - 1

end_space = tree_height - 1

hashes = 1

while tree_height != 0:
	
	for i in range(space):
		print( " ", end="")
	
	space -= 1
	
	for j in range( hashes ):
		print("#", end="")
	
	hashes += 2
	
	tree_height -= 1
	print()

for k in range(end_space):
	print(" ", end="")
print ("#")
